bmdbutils/biomodelos/biomodelos.py

```diff
@@ -49,3 +49,18 @@ def query_editions(self, tax_ids, init_date, end_date):
             """.format(init_date=init_date, end_date=end_date, tax_ids=",".join(tax_ids))
         editions = pd.read_sql_query(query, cnx)
         return editions
+    
+    def query_ecovars(self, tax_ids, init_date, end_date):
+        cnx = create_engine('postgresql://{username}:{password}@{addr}:{port}/{dbname}'.format(
+            username=self.pg_user, password=quote_plus(self.pg_pass), addr=self.pg_addr, port=self.pg_port, dbname=self.pg_db))
+        query = """SELECT evs.user_id, u.name username, evs.species_id, ev.name ecological_variable, evs.created_at, evs.updated_at
```

Comment:
I think you have an extra `username` here

Reply:
Fixed